mslib/netCDF4tools.py

```diff
@@ -254,8 +254,8 @@ def __init__(self, files, exclude=None, skip_dim_check=None,
         masterDims = list(cdfm.dimensions.keys())
         # Check that each dimension has a coordinate dimension
         for dimName in masterDims:
-            if dimName not in cdfm.variables:
-                raise IOError(f"dimension '{dimName}' has no coordinate variable in master '{master}'")
+            if dimName not in cdfm.variables and dimName not in skip_dim_check:
+                raise IOError("dimension '{}' has no coordinate variable in master '{}'".format(dimName, master))
```

Comment:
Was this commit supposed to revert to the .format notation?

Reply:
nope, I correct it, guess I have not selected the best solution.